### ISSUE TITLE
Issue on page /intro/abstractions.html

### ISSUE THREAD

**Issue by krish-shahh** [OpenOSOrg/openos#9]
For the images on the page, if possible please make them modals with an "X" button because when you click on them, it opens the image on the page with no way to get back to the other content of the page other than the back arrow button on the browser. You can make the modal when the image is clicked on and you can exit the modal either through an "X" button or using the escape key on your keyboard. Again, this is just a suggestion for better user interaction. Thanks! 

https://www.w3schools.com/howto/howto_css_modal_images.asp

**Comment by okrieg** [comment]
This sounds like a good idea, but do you know how to get modal images in jupyter book?

**Comment by krish-shahh** [comment]
> This sounds like a good idea, but do you know how to get modal images in jupyter book?

Yes, you have to utilize the IPython.display module in Jupyter. If you would like, I can help create an implementation for this.

**Comment by okrieg** [comment]
Just confirming, this is not something we can do?

**Comment by krish-shahh** [comment]
> Just confirming, this is not something we can do?

As of now, it is not possible since the textbook utilizes myst markdown for the images, and opening the images in a pop-up modal is not supported in myst markdown.
